steering_vectors/aggregators.py

```diff
@@ -7,7 +7,6 @@
 
 
 Aggregator = Callable[[Tensor, Tensor], Tensor]
-Regression = Union[LinearRegression, LogisticRegression]
```

Comment:
Sorry I'm an idiot I told you to remove this because I thought it wasn't being used but I missed that it's used in `_get_normalized_regression_coef()` 🤦‍♂️ . Really sorry - feel free to add this back if you want!

Reply:
I thought you meant to remove it since the type is only used in one place. I don't think it's necessary to add it back now, but would do so if we start using the union elsewhere